tests/scripts/thread-cert/border_router/test_dnssd_instance_name_with_space.py

```diff
@@ -115,6 +115,7 @@ def test(self):
 
         self._verify_service_browse_result(client.dns_browse(SERVICE_FULL_NAME, server=br1.get_rloc()))
         self._verify_service_browse_result(client.dns_browse(SERVICE_FULL_NAME, server=br2.get_rloc()))
+        self._verify_service_browse_result(client.dns_browse(SERVICE_FULL_NAME.lower(), server=br2.get_rloc()))
```

Comment:
What is the purpose of this? Is `SERVICE_FULL_NAME` already all lower case?

Reply:
```suggestion
        self._verify_service_browse_result(client.dns_browse(SERVICE_FULL_NAME.upper(), server=br2.get_rloc()))
```